hazelcast/src/test/java/com/hazelcast/map/SizeEstimatorTest.java

```diff
@@ -57,7 +57,7 @@ public void testSinglePut() throws InterruptedException {
 
     @Test
     public void testExactHeapCostAfterUpdateWithMultipleBackupNodes() throws InterruptedException {
-        long expectedPerEntryHeapCost = 172L;
+        long expectedPerEntryHeapCost = 156L;
```

Comment:
it is here https://github.com/hazelcast/hazelcast/commit/b90f8dbd9ebac29a02ecfe6c6f44bc1a1e62269d#diff-cddfa764e142a56c228ed23a9802d1a7R35